src/i18n.js

```diff
@@ -22,18 +22,32 @@ import sv from 'i18next-icu/locale-data/sv'
 import uk from 'i18next-icu/locale-data/uk'
 import zh from 'i18next-icu/locale-data/zh'
 
+const localeData = [cs, da, de, en, eo, es, fr, it, ko, nl, no, pl, pt, ru, sl, sv, uk, zh]
+
+export const localesList =
+  // add here the language variants
+  ['ko-KR', 'zh-CN', 'zh-HK', 'zh-TW']
+    .concat(localeData.map((locale) => locale[0].locale))
+    // add here languages you want to exclude
+    .filter(item => !['zh'].includes(item))
+    .filter(item => !['ko'].includes(item))
+    .sort()
+
 i18n
-  .use(new ICU({
-    localeData: [cs, da, de, en, eo, es, fr, it, ko, nl, no, pl, pt, ru, sl, sv, uk, zh]
-  }))
+  .use(new ICU({ localeData: localeData }))
   .use(XHR)
   .use(LanguageDetector)
   .init({
     ns: ['welcome', 'status', 'files', 'explore', 'peers', 'settings', 'notify'],
-    fallbackLng: 'en',
+    fallbackLng: {
+      'zh-HANS': ['zh-CN', 'en'],
```

Comment:
After much digging, i think `Hans` and `Hant` should only have the first letter capitalized

```suggestion
      'zh-Hans': ['zh-CN', 'en'],
```

Reply:
See:
- Mozilla langmap https://github.com/mozilla/language-mapping-list/blob/6a1ba99f9625486705f0872750df228f52498bea/language-mapping-list.js#L837
- Spec http://www.ietf.org/rfc/bcp/bcp47.txt